### PR TITLE
New cmdlet Set-AzSqlInstanceDatabase

### PR DESCRIPTION
**Pull request by milanbrkic-ms**
## Description

Adding new command Set-AzSqlInstanceDatabase to enable updates on managed database.

## Checklist

- [X] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [X] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Sql/Sql/ManagedDatabase/Cmdlet/SetAzureSqlManagedDatabase.cs [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+using System.Collections;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedDatabase.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to create a new Azure Sql Managed Database
+    /// </summary>
+    [Cmdlet(VerbsCommon.Set, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceDatabase",
+        DefaultParameterSetName = SetByNameAndResourceGroupParameterSet,
+        SupportsShouldProcess = true),
```
[the remark this answers is not in the document]
Comment (MilanBrkicFON) [reply src/Sql/Sql/ManagedDatabase/Cmdlet/SetAzureSqlManagedDatabase.cs:31]: It returns tags in the response object when used -WhatIf

File: src/Sql/Sql/ManagedDatabase/Services/AzureSqlManagedDatabaseAdapter.cs [hunk 2]
```diff
@@ -103,6 +103,23 @@ internal AzureSqlManagedDatabaseModel UpsertManagedDatabase(string resourceGroup
...
+        /// </summary>
+        /// <param name="resourceGroup">The name of the resource group</param>
+        /// <param name="managedInstanceName">The name of the Azure Sql Database Managed Instance</param>
+        /// <param name="model">The input parameters for the create/update operation</param>
+        /// <returns>The upserted Azure Sql Database from AutoRest SDK</returns>
+        internal AzureSqlManagedDatabaseModel UpdateManagedDatabase(string resourceGroup, string managedInstanceName, AzureSqlManagedDatabaseModel model)
+        {
+            var resp = Communicator.Update(resourceGroup, managedInstanceName, model.Name, new Management.Sql.Models.ManagedDatabaseUpdate
+            {
+                Tags = model.Tags,
```
[the remark this answers is not in the document]
Comment (MilanBrkicFON) [reply src/Sql/Sql/ManagedDatabase/Services/AzureSqlManagedDatabaseAdapter.cs]: Currently, no. But I guess we would add more and more

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**